index.html

```
@@ -0,0 +1,3 @@
+<p> hello </p>
```

HTMLHint (1.9.2)

[error] Doctype must be declared before any non-comment content.

(doctype-first)